lib/miq_automation_engine/engine/miq_ae_engine/drb_remote_invoker.rb

```diff
@@ -148,6 +152,13 @@ def backtrace_with_evm
     value ? filter_backtrace(value) : value
   end
 
+  def get_file_info(line)
+    LINE_HASH.each do |fqname, item|
+      return fqname, line - item[:start] if line.between?(item[:start], item[:end])
+    end
+    return "?", line
```

Comment:
Previously, this would be RUBY_METHOD_NAME, so I think that would be preferable than `?`